vendor/bou.ke/monkey/LICENSE.md

```diff
@@ -0,0 +1,5 @@
+Copyright Bouke van der Bijl
+
+I do not give anyone permissions to use this tool for any purpose. Don't use it.
+
+I’m not interested in changing this license. Please don’t ask. 
```

Comment:
hm, this is a show-stopper

Reply:
Good catch, this library was only being used in the unit test to patch out a method. I just pushed a small refactor to remove this dependency.